docs/culture/psychological-safety.md

```diff
@@ -0,0 +1,107 @@
+* "This team I am in is all over the map! If this were my lab, I would be telling more people what to do."
+
+**Questions for Reflection**
+* How does my position in the academic hierarchy make me feel about myself?
+* What career stage are others at who are around me? How does this compare with my position?
+* Given our relative positions, what kind of dynamic do I think this sets up in the room?
+:::
+
+## Hackweek Examples
+
+We have prepared two videos that simulate typical scenarios occurring at a hackweek. (more to come...)
+
+## Group Activity
+
+We invite you to reflect on your own identity and how this has shaped your experience so far as an academic. Scholars typically use some type of circle or wheel to visualize intersecting identities and their interplay with power and privilege {cite:p}`morgan_describing_1996`. We'll work with a version of such a wheel that locates high privilege in the center transitioning to lower privilege towards the outer edge of the circle.
+
+:::{admonition} Activity Instructions
+:class: tip
+The purpose of this activity is to locate yourself on your own wheel of power and privilege. We will not ask you to share your wheel with others. Instead, we'll invite you to discuss in small groups general ways your experience of the exercise might affect how you show up at the hackweek. 
+1. Think about 3 or 4 identities that have particular meaning for you when you think about participating in a learning environment, and jot these down in a list.
+1. Take the image we have provided of a circle with the word "high privilege" at the center, and "low privilege" on the outer edge.
+1. Draw a line from the center of the circle to the outer edge for each of the identities you described. 
+1. Choose a word that represents one realization of that identity associated with high and low privilege, and label your diagram accordingly.
+1. Now, draw a star on each of your identity lines showing where you place yourself on the continuum of privilege.
+1. Optionally, connect each of your stars with a line. The total area inscribed represents your overall privilege with respect to the identities you chose.
+
+**Prompts for Discussion**
+* Typically the center of such a circle is associated with greater power, and the outer edges associated with marginalization (see example [wheel of power and privilege](https://usq.pressbooks.pub/app/uploads/sites/53/2022/05/Power.jpg)). Does this ring true for your example?
+* (other prompts)
+:::
+
+## Suggested "hacks" to build awareness 
+
+The overall goal of this module is for us to begin building awareness of our own identities and those of others in the context of systemic patterns of power, privilege and marginalization. As our awareness grows we can begin altering our own behaviors to continue fostering psychological safety in group settings. Here are some suggested things to ask ourselves when entering any learning space:
```

Comment:
This is great! Could consider moving the overall goal statement in the first sentence to the very top of the page